mobile/src/card-view.js

```diff
@@ -305,7 +313,12 @@ export class CardView extends Component {
 }
 
 export class CardListView extends Component {
-  state = { notes: this.props.notes || '', isFocused: false, inputHeight: 0, isEdit: false }
+  state = { notes: this.props.notes || '', isFocused: false, inputHeight: 0, isEdit: false, isDisabled: null}
+
+  componentDidMount(){
+    client.getAttendee(this.props.user.id).then(user => this.setState({isDisabled: false}))
```

Comment:
This will trigger a network request for each attendee in the list.  Is this necessary?

Reply:
This was to resolve a ticket for a deleted user going to a blank profile page. I agree its unnecessary

Reply:
I agree. Getting a rare blank profile page is better than sending all this network traffic each time.  Let's revert this fix and mark this as "As Designed"